src/v/coproc/event_listener.cc

```diff
@@ -29,6 +29,18 @@
 
 #include <exception>
 
+namespace {
+
+kafka::client::client make_client() {
```

Comment:
does static work as fine as an anonymous namespace?

Reply:
i think they are equivalent for creating local scope stuff. this is probably more idiomatic c++